DOM_creation/Lesson3/main.js

```diff
@@ -0,0 +1,22 @@
+const ul = document.getElementById("js-ul");
+
+const a1 = document.createElement("a");
+a1.href = "a1.html";
+
+const a2 = document.createElement("a");
+a2.href = "a2.html";
+
+const img1 = document.createElement("img");
+img1.src = "/img/bookmark.png";
+
+const img2 = document.createElement("img");
+img2.src = "/img/message.png";
+
+const array = [{a: a1, img: img1, text: "a1"}, {a: a2, img: img2, text: "a2"}];
```

Comment:
[IMO]
You can change the variable name `array` to something which describes the contents of the variable (e.g. links). It makes more sense and makes your code easier to understand, rather than just calling it an 'array'.

Then when you iterate through each element using forEach or map, you can use a singular form for the first argument. In your case, it'll look like this:

```
links.forEach(link => {
```

The following page explains the concept in more detail:
[JavaScript forEach - the complete guide](https://blog.learnjavascript.online/posts/javascript-foreach-the-complete-guide/)

Reply:
Thank you for your reminder Kayano-san,
It is very fundamental and important but I tend to forget this rule of thumb to name variables. 
Thank you for sharing the link as well!